presto-main/src/main/java/com/facebook/presto/execution/resourceGroups/InternalResourceGroupManager.java

```diff
@@ -284,6 +287,12 @@ public List<ResourceGroupRuntimeInfo> getResourceGroupRuntimeInfos()
         return resourceGroupRuntimeInfos.build();
     }
 
+    @Override
+    public boolean configurationManagerLoaded()
```

Comment:
make the method name `configurationManagerLoaded` -> `isConfigurationManagerLoaded`. similar to the variable name.

Reply:
method name updated.